scripts/deploy-base

```diff
@@ -19,10 +18,10 @@ else
 	exit 1;
 fi
 
-echo "Deploying APM registry to aragonpm.eth"
-npx truffle exec --network rpc scripts/deploy-apm.js
+echo "Deploying APM registry to aragonpm.eth and open.aragonpm.eth"
+npx truffle exec --network rpc scripts/deploy-apm-open.js
 
-cd ../aragon-id;
+cd repos/aragon-id;
 echo "Deploying aragonID registry to aragonpm.eth"
 npx truffle compile;
 npx truffle exec --network rpc scripts/deploy-beta-aragonid.js;
```

Comment:
Now I'm starting to wonder if we should move the aragonID deployment script to here as well and just use `@aragon/id` as a dependency.

Reply:
Make sense , so we have the same workflow for both